src/PureCloud.Client/Models/AnalyticsQueryResponse.cs

```diff
@@ -0,0 +1,7 @@
+namespace PureCloud.Client.Models;
+
+public sealed class AnalyticsQueryResponse
```

Comment:
Why creating new classes instead of reusing the old models we had? All need to happen is clean up the old
Models and include them in the project. It's safer migration approach than creating new models